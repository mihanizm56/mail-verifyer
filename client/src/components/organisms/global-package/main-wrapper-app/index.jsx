import React, { memo } from "react";
import { MainRouterLayout } from "../../../../routes";
import "./MainWrapper.css";

export const MainWrapper = memo(props => (
	<div className="main-wrapper">
		<MainRouterLayout />
	</div>
));
