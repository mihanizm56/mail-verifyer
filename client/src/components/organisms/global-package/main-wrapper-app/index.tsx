import React, { memo } from "react";
import { MainRouterLayout } from "../../../../routes";
import "./MainWrapper.css";

export const MainWrapper = memo(() => (
	<div className="main-wrapper">
		<MainRouterLayout />
	</div>
));
