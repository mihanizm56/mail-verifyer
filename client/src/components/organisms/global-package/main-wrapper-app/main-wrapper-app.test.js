import React, { createElement } from "react";
import { shallow } from "enzyme";
import { MainWrapper } from "../main-wrapper-app";

describe("test MainWrapper", () => {
	describe("test render contract", () => {
		it("render snap with all props", () => {
			const component = shallow(<MainWrapper />);

			expect(component).toMatchSnapshot();
		});
		it("render MainRouterLayout module", () => {
			const component = shallow(<MainWrapper />);
			const Sender = component.find("MainRouterLayout");

			expect(Sender.length).toEqual(1);
		});
	});
});
