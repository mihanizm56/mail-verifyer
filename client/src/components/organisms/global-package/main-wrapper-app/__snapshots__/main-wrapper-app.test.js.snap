// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test MainWrapper test render contract render snap with all props 1`] = `
<div
  className="main-wrapper"
>
  <MainRouterLayout />
</div>
`;
